includes/definitions/discovery/omnipcx.yaml

```diff
@@ -9,7 +9,7 @@ modules:
                     num_oid: '.1.3.6.1.4.1.637.64.4400.1.2'
                     descr: 'Pbx State'
                     state_name: pbxState
-                    index: 'pbxState'
+                    index: 'pbxState{{ $index }}'
```

Comment:
Hi @martijn-schmidt 
Why did you change other YAML files than the one impacted by your change ?  PBX state for instance is a "standalone" oid, the $index value has no particular meaning here.